src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -92,8 +93,12 @@ public function getTranslationNodeVisitor(): TranslationNodeVisitor
         return $this->translationNodeVisitor ?: $this->translationNodeVisitor = new TranslationNodeVisitor();
     }
 
-    public function trans(string $message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
+    public function trans($message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
     {
+        if (!\is_string($message) && !$message instanceof TranslatableInterface) {
+            throw new \Exception('Message passed must by of the type string or implement '.TranslatableInterface::class);
```

Comment:
phpdoc serves as a contract, no need for safe guards here. (in PHP8 we have a true safe guard with unions)

Reply:
also this breaks passing __toString objects

Reply:
This was mostly to provide a nicer error with twig. Since we don't have an interface, it's likely needed less than before